src/it/scala/com/ocadotechnology/pass4s/connectors/jms/JmsRecoveryTests.scala

```diff
@@ -1,7 +1,7 @@
 package com.ocadotechnology.pass4s.connectors.jms
 
-import akka.actor.ActorSystem
-import akka.stream.alpakka.jms.ConnectionRetryException
+import org.apache.pekko.actor.ActorSystem
```

Comment:
Why those tests were originally not in akka module?

Reply:
Yup, I've restored them now.